addons/cargo/functions/fnc_addCargoItem.sqf

```diff
@@ -26,4 +26,4 @@ for "_i" from 1 to _amount do {
 };
 
 // Invoke listenable event
-["ace_cargoAddedByClass", [_itemClass, _vehicle, _amount]] call CBA_fnc_globalEvent;
+["ace_cargoByClassAdded", [_itemClass, _vehicle, _amount]] call CBA_fnc_globalEvent;
```

Comment:
Originally this even fired only when cargo was added by providing a classname, not sure if that's still the same though.